IM_calculation/Advanced_IM/advanced_IM_factory.py

```diff
@@ -1,12 +1,16 @@
+from pathlib import Path
 from collections import namedtuple
+
 import pandas as pd
 import os
 import re
 import subprocess
 import tempfile
 
-from qcore.timeseries import seis2txt
+from qcore.timeseries import timeseries_to_text
 from qcore.utils import load_yaml
```

Comment:
Not a you problem, but this `load_yaml` function is completely unnecessary now that Python has standardised the behaviour that `dict` keys are always ordered (indeed, `OrderedDict` has a completely different purpose now that `load_yaml` is almost certainly not considering properly). You can without loss of generality replace all instance of `load_yaml` with `yaml.safe_load`.

Reply:
We could make this an issue for the repository, more of a code clean up task than actually fixing anything unless it hindered performance for example which in this case I don't think so, so I'd say this is pretty low priority.